documentation/source/library_reference.rst

```diff
@@ -317,6 +315,22 @@ XGMII
 Miscellaneous
 =============
 
+Cocotb Run Information
```

Comment:
Hmm, we don't usually capitalize cocotb. At any rate, it seems sort of redundant.

Perhaps `Simulator runtime information`?

Reply:
Well it's not just simulator information. `Miscellaneous Runtime Information`?

Reply:
New heading looks ok.